player.js

```diff
@@ -0,0 +1,66 @@
+var Scrabble = require("./scrabble.js");
+
+var Player = function(name) {
+  this.name = name;
+  this.played_words = [];
+  this.score = 0
+}
+
+Player.prototype.plays = function () {
+  return this.played_words;
+}
+
+Player.prototype.play = function (word) {
+  if (this.hasWon() === true) {
+    return false
+  }
+  if (typeof word === "object") {
+    for (var i = 0; i < word.length; i++) {
+      this.played_words.push(word[i]);
+    }
+  }
+  else {
+    this.played_words.push(word);
+  }
+  this.total_score()
+  return this.plays();
+}
+
+Player.prototype.total_score = function () {
+  var scrabble = new Scrabble();
+  var sum = 0
+  for (var i = 0; i < this.played_words.length; i++) {
+    sum += scrabble.score(this.played_words[i]);
+  }
+  this.score = sum;
+  return this.score;
+}
+
```

Comment:
Hi Amy,

Thanks for the feedback, after I finish my projects, I will start looking
at my code to make sure I can clean it up and such.  I really like the code
reviews, I appreciate it so much, it's nice to hear how my code is going!

Adriana

On Tue, Jun 21, 2016 at 11:03 AM, Amy Hunter <notifications@github.com>
wrote:

> In player.js
> <https://github.com/Ada-C5/js-scrabble/pull/20#discussion_r67920323>:
>
> > +    this.played_words.push(word);
> > +  }
> > +  this.total_score()
> > +  return this.plays();
> > +}
> > +
> > +Player.prototype.total_score = function () {
> > +  var scrabble = new Scrabble();
> > +  var sum = 0
> > +  for (var i = 0; i < this.played_words.length; i++) {
> > +    sum += scrabble.score(this.played_words[i]);
> > +  }
> > +  this.score = sum;
> > +  return this.score;
> > +}
> > +
>
> A lot of these if/else statements can be rewritten to use ternary
> operators, so the following code would look like this:
> if this.score >= 100 ? true : false;
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Ada-C5/js-scrabble/pull/20/files/6246f16b06ec1bdfef038a2ac63baafc685feefd#r67920323>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AOVCTWh9PouSL-FnWiungngHKXuxV0Kiks5qOCd1gaJpZM4Iz--j>
> .
>